src/SocketLabs/InjectionApi/SendResponse.cs

```diff
@@ -98,7 +98,7 @@ public string ResponseMessage
                         return "No valid message body was found in the message";
 
                     case SendResult.InvalidTemplateId:
-                        return "An invalid TemplateId was found in the message";
+                        return "MetadataOrTagsAreTooLarge => “Metadata and tags exceed 12.5KB";
```

Comment:
Not the correct change. Please revert this.

A new `SendResult` needs to be added for the error and a new case is needed in the switch to handle it.